backend/src/main/java/com/cgi/retrospecto/backend/poker/controller/impl/PokerControllerImpl.java

```diff
@@ -0,0 +1,90 @@
+package com.cgi.retrospecto.backend.poker.controller.impl;
+
+import com.cgi.retrospecto.backend.poker.api.PokerConstants;
+import com.cgi.retrospecto.backend.poker.controller.PokerController;
+import com.cgi.retrospecto.backend.poker.controller.dto.*;
+import com.cgi.retrospecto.backend.poker.domain.Room;
+import com.cgi.retrospecto.backend.poker.domain.Story;
+import com.cgi.retrospecto.backend.poker.domain.Vote;
+import com.cgi.retrospecto.backend.poker.exception.RoomNotFoundException;
+import com.cgi.retrospecto.backend.poker.exception.StoryNotFoundException;
+import com.cgi.retrospecto.backend.poker.helper.converter.RoomConverter;
+import com.cgi.retrospecto.backend.poker.helper.converter.StoryConverter;
+import com.cgi.retrospecto.backend.poker.helper.converter.VoteConverter;
+import com.cgi.retrospecto.backend.poker.service.RoomService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.web.bind.annotation.*;
+
+import static com.cgi.retrospecto.backend.poker.api.PokerConstants.ApiConstants.POKER;
+
+@RestController
+@RequestMapping(POKER)
+@CrossOrigin
+public class PokerControllerImpl implements PokerController {
+    @Autowired
+    private RoomService roomService;
+
+    // @formatter:off
+    @Override
+    public ResponseEntity<RoomDTO> createRoom(@RequestBody CreateRoomDTO dto) {
+        return new ResponseEntity<>(
+                RoomConverter.toDTO(
+                    roomService.createRoom(
+                            RoomConverter.toEntity(dto, new Room())
```

Comment:
The parameter new Room() could be deleted and just used in the toEntity method, this would keep the signature lighter.

Reply:
Removed.